Source/Examples/WPF.SharpDX/SimpleDemo/MainViewModel.cs

```diff
@@ -40,15 +40,14 @@ public class MainViewModel : BaseViewModel
         public Color4 DirectionalLightColor { get; private set; }
         public Color4 AmbientLightColor { get; private set; }
 
-
         public MainViewModel()
         {
             // titles
-            this.Title = "Simple Demo";

```

Comment:
I think it is OK to continue with no coding style rules on HelixToolkit.Wpf.SharpDX. I don't think it is a problem that the style is not consistent across the projects. I can add a note about this somewhere!